packages/website/pages/docs/client/generated.md

```diff
@@ -0,0 +1,19 @@
+# OpenAPI generated clients
+
+If you're integrating NFT.Storage with a JavaScript project, the [JavaScript client][reference-js-client] is a natural fit. If JavaScript isn't part of your stack, the NFT.Storage [HTTP API][reference-http-api] can be used from any programming language capable of making HTTP requests. 
+
+In addition to using the standard request libraries for your language, you can use client libraries generated from the [OpenAPI](https://www.openapis.org/) schema that describes the API.
```

Comment:
Should we point out here that JS client has a custom logic for chunking uploads and that generated clients will be limited to 100Mb uploads?

Reply:
Yeah, that's a good call out 👍